### PR TITLE
Make icons appear again

### PR DESCRIPTION
**Pull request by ernestask**
Takes care of https://github.com/abrt/gnome-abrt/issues/226 on the gnome-abrt side.

This one is a bit difficult to split up, because of how coupled the D-Bus implementation was with command-line handling.

### CODE REVIEW

**Comment by ernestask** [comment]
If someone complains about whatever was fixed by the code I removed, we can bring it back, but I honestly can’t tell what the problem was, because no one bothered to actually explain anything.

File: src/gnome-abrt [hunk 1]
```diff
@@ -189,6 +100,33 @@ class OopsApplication(Gtk.Application):
...
+        '''
+        // Translators: This is a description of --problem command line option
+        // displayed when a user runs: `gnome-abrt --help'
+        _("Selected problem ID")
+        '''
+        self.add_main_option('problem',
+                             ord('p'),
+                             GLib.OptionFlags.NONE,
+                             GLib.OptionArg.STRING,
+                             _('Be verbose'),
```
Comment (mkutlak) [inline src/gnome-abrt]: Too verbose. 

`s/'Be verbose'/'Selected problem ID'/`

**Comment by mkutlak** [comment]
Can you fix these pylint warnings?
```
************* Module gnome-abrt
gnome-abrt:167: [W0621(redefined-outer-name), OopsApplication.do_command_line] Redefining name 'configuration' from outer scope (line 335)
gnome-abrt:165: [W0221(arguments-differ), OopsApplication.do_command_line] Parameters differ from overridden 'do_command_line' method
gnome-abrt:335: [C0103(invalid-name), ] Constant name "configuration" doesn't conform to '(([A-Z_][A-Z1-9_]*)|(__.*__))$' pattern
gnome-abrt:338: [C0103(invalid-name), ] Constant name "application" doesn't conform to '(([A-Z_][A-Z1-9_]*)|(__.*__))$' pattern
gnome-abrt:26: [W0611(unused-import), ] Unused ArgumentParser imported from argparse
gnome-abrt:77: [W0611(unused-import), ] Unused import dbus
gnome-abrt:79: [W0611(unused-import), ] Unused import dbus.service
gnome-abrt:81: [W0611(unused-import), ] Unused DBusGMainLoop imported from dbus.mainloop.glib
```

File: src/gnome-abrt (hunk 1 above)
Comment (ernestask) [reply src/gnome-abrt]: I don’t know, I feel like we undersell our Verbose Output® technology.

**Comment by ernestask** [comment]
Absolutely I can.

**Comment by ernestask** [comment]
Should be 10/10 now.

**Comment by mkutlak** [comment]
@ernestask can you resolve the conflicts?

**Comment by ernestask** [comment]
Done.

**Pull request merged**